RussianFilter/sections/css_extended.txt

```diff
@@ -203,6 +203,8 @@ zhzh.com.ua#?#.content > .site-name-s > h2:contains(Новини Житомир
 zhzh.com.ua#?#.aside-block:has(.adsbygoogle)
 novorosinform.org#?#.td-ss-main-sidebar .widget_text:has(> h4 > span:contains(/Новости партнеров|Актуально/))
 rgmechanics.ru#?#div:has(> center > div[id^="advertur_"])
+pogoda.ngs.ru##.article-item:has(a[href^="https://n1.ru"])
+pogoda.ngs.ru##.article-item:has(a[href*="zarplata.ru"])
```

Comment:
Ext-CSS is not required here, those are the two last elements and even if they would be not the last elements, only a very small left-over would be shown.

Reply:
So, what I need to do with them? If i need to block this blocks can u show me an example?